portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,25 +32,62 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
-  @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    ArrayList<String> micaylaMessages = new ArrayList<>();
-    micaylaMessages.add("My name is Micayla.");
-    micaylaMessages.add("I am from the Bay Area.");
-    micaylaMessages.add("I live in New York City.");
+  ArrayList<String> comments = new ArrayList<>();
```

Comment:
so one of goals of using the datastore is to avoid having to store comments in the server's memory. Let's remove this.

Reply:
Hmmm, not sure why this comment was resolved, given that the local array is still here...